### PR TITLE
refine jepsen tests

### PR DESCRIPTION
**Pull request by yito88**
- Acquire Scalar DB from maven
- Support Jepsen CLI
- Renamed the Jepsen directory

### CODE REVIEW

File: jepsen/scalardb/README.md [hunk 1]
```diff
@@ -61,8 +60,8 @@ RUN cd /jepsen/cassandra && lein install
...
 # in jepsen-control
 
-cd /jepsen/scalar
-lein test :only scalar.transfer-test/transfer-bridge
+$ cd /jepsen/scalardb
+$ lein run test --test transfer --nemesis crash --join decommission --time-limit 300
 ```
 
-  - You can find the list of tests in `/jepsen/scalar/test/scalar/transfer_test.clj`
+  - You can find full options by `lein run test --help`
```
Comment (iamcap7) [inline jepsen/scalardb/README.md]:
```suggestion
  - Use `lein run test --help` to see a list of the full options
```

Comment (yito88) [reply jepsen/scalardb/README.md]: Thanks!

File: jepsen/scalardb/README.md [hunk 2]
```diff
@@ -9,43 +9,42 @@ The current Scalar DB tests work with [Cassandra test tools in Jepsen](https://g
 1. Get Jepsen which has Cassandra tests
```
Comment (feeblefakie) [inline jepsen/scalardb/README.md]: Can you update the first section ?
```
# Jepsen tests for Scalar DB

This guide will teach you how to run Jepsen tests for Scalar DB.
The current tests use [Cassandra test tools in Jepsen](https://github.com/scalar-labs/jepsen/tree/cassandra).
```
@iamcap7  Please correct my english as needed.

Comment (iamcap7) [reply jepsen/scalardb/README.md]: @feeblefakie It is not needed 👍

Comment (yito88) [reply jepsen/scalardb/README.md]: Thanks. Done

**Pull request merged**